src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerProcessor.cs

```diff
@@ -53,8 +56,8 @@ public virtual void EnqueueMessage(LogMessageEntry message)
         // for testing
         internal virtual void WriteMessage(LogMessageEntry entry)
         {
-            IConsole console = entry.LogAsError ? ErrorConsole : Console;
-            console.Write(entry.Message);
+            IConsole? console = entry.LogAsError ? ErrorConsole : Console;
```

Comment:
seems like ErrorConsole and Console are getting set in the ConsoleLoggerProvider right after ConsoleLoggerProcessor gets instantiated. Maybe in a next PR we could update the ConsoleLoggerProcessor itself to set ErrorConsole and Console so this `console` assignment becomes improved and you wouldn't need `?` or `!`